x-pack/plugins/enterprise_search/server/lib/enterprise_search_config_api.ts

```diff
@@ -67,44 +67,60 @@ export const callEnterpriseSearchConfigAPI = async ({
+            totalFields:
+              data?.settings?.configured_limits?.workplace_search?.custom_api_source?.total_fields,
+          },
+        },
       },
       appSearch: {
-        accountId: data?.settings?.app_search?.account_id,
-        onBoardingComplete: !!data?.settings?.app_search?.onboarding_complete,
+        accountId: data?.current_user?.app_search?.account?.id,
+        onBoardingComplete: !!data?.current_user?.app_search?.account?.onboarding_complete,
         role: {
-          id: data?.current_user?.app_search_role?.id,
-          roleType: data?.current_user?.app_search_role?.role_type,
+          id: data?.current_user?.app_search?.role?.id,
+          roleType: data?.current_user?.app_search?.role?.roleType,
           ability: {
-            accessAllEngines: !!data?.current_user?.app_search_role?.ability?.access_all_engines,
-            destroy: data?.current_user?.app_search_role?.ability?.destroy || [],
-            manage: data?.current_user?.app_search_role?.ability?.manage || [],
-            edit: data?.current_user?.app_search_role?.ability?.edit || [],
-            view: data?.current_user?.app_search_role?.ability?.view || [],
-            credentialTypes: data?.current_user?.app_search_role?.ability?.credential_types || [],
+            accessAllEngines: !!data?.current_user?.app_search?.role?.ability?.access_all_engines,
+            destroy: data?.current_user?.app_search?.role?.ability?.destroy || [],
+            manage: data?.current_user?.app_search?.role?.ability?.manage || [],
+            edit: data?.current_user?.app_search?.role?.ability?.edit || [],
+            view: data?.current_user?.app_search?.role?.ability?.view || [],
+            credentialTypes: data?.current_user?.app_search?.role?.ability?.credential_types || [],
             availableRoleTypes:
-              data?.current_user?.app_search_role?.ability?.available_role_types || [],
+              data?.current_user?.app_search?.role?.ability?.available_role_types || [],
           },
         },
       },
       workplaceSearch: {
-        canCreateInvitations: !!data?.settings?.workplace_search?.can_create_invitations,
-        isFederatedAuth: !!data?.settings?.workplace_search?.is_federated_auth,
         organization: {
-          name: data?.settings?.workplace_search?.organization?.name,
-          defaultOrgName: data?.settings?.workplace_search?.organization?.default_org_name,
+          name: data?.current_user?.workplace_search?.organization?.name,
+          defaultOrgName: data?.current_user?.workplace_search?.organization?.default_org_name,
         },
-        fpAccount: {
-          id: data?.settings?.workplace_search?.fp_account.id,
-          groups: data?.settings?.workplace_search?.fp_account.groups || [],
-          isAdmin: !!data?.settings?.workplace_search?.fp_account?.is_admin,
-          canCreatePersonalSources: !!data?.settings?.workplace_search?.fp_account
+        account: {
```

Comment:
@scottybollinger - FYI, Brian renamed `fpAccount` to just `account` and I followed suit, mostly because I think the distinction is no longer really necessary in the new Kibana world (theoretically since it's client-facing only, it shouldn't need to hold references to our old code names).

That being said, this change only affects the initial data/config endpoint & doesn't preclude you from using `fpAccount` in the rest of Workplace Search if your team strongly prefers to do so - just a thought that you could probably clean that var name up moving forward if you wanted to.